apps/starter/src/ensemble/screens/askEmbrace.yaml

```diff
@@ -16,12 +16,45 @@ View:
   body:
     Column:
       children:
-        # - Markdown:
-        #     id: askText
-        #     text: "## Ask Embrace"
-        # - Text:
-        #     id: he
-        #     text: hehe
+        - Markdown:
+            id: askText
+            text: "### Toggle Buttons"
+
+        - ToggleButton:
+            id: myToggleButton
+            styles:
+              margin: 5
+              spacing: 10
+              runSpacing: 3
+              color: black
+              selectedColor: yellow
+              backgroundColor: white
+              selectedBackgroundColor: pink
+              selectedBorderColor: blue
+              borderColor: green
+            value: red
+            items:
+              # - red
+              # - green
+              # - blue
+              # - yellow
+              - { "label": "Red", "value": "red", "icon": { "name": "Star" } }
```

Comment:
This is equivalent to specifying a YAML object correct? This should still work if you have

```ts
- label: Red
  value: red
  icon:
    name: Star
```

Reply:
yes